vic/drivers/image/include/vic_driver_image.h

```diff
@@ -63,6 +70,7 @@ typedef struct {
     size_t n_nx; /**< size of x-index; */
     size_t n_ny; /**< size of y-index */
     location_struct *locations; /**< locations structs for local domain */
+    location_grid_struct *locations_grid; /**< locations structs for full unmasked latlon domain */
```

Comment:
That is possible, but keep in mind that in that case the indexes are changing. I mean the following:
In the case you suggest, `location_struct` is bigger because the unmasked gridcells are now included. This means that `location_struct[n]` refers to another lonlat combination. We have to be very careful that this will not effect other parts of the VIC code.
What shall we do now to solve it?